staging/src/k8s.io/component-base/metrics/prometheus/restclient/metrics.go

```diff
@@ -40,6 +40,26 @@ var (
 		[]string{"verb", "host"},
 	)
 
+	requestSize = k8smetrics.NewHistogramVec(
+		&k8smetrics.HistogramOpts{
+			Name: "rest_client_request_size_bytes",
+			Help: "Request size in bytes. Broken down by verb and host.",
+			// 64 bytes to 16MB
+			Buckets: []float64{64, 256, 512, 1024, 4096, 16384, 65536, 262144, 1048576, 4194304, 16777216},
```

Comment:
you forgot the stability level.

Reply:
it seems it's missing on the older metric above, can you add it there too?

Reply:
ups

Reply:
yeah, I've added it on both